scripts/asr/transcribe_file.py

```diff
@@ -73,6 +73,7 @@ def main() -> None:
             encoding=riva.client.AudioEncoding.LINEAR_PCM,
             language_code=args.language_code,
             max_alternatives=1,
+            profanity_filter=False,
```

Comment:
What's the point of adding the profanity_filter parameter if we hard-code it to be false?